File: routeMap/app/src/main/java/ru/android/zheka/di/HomeBinding.java
package ru.android.zheka.di;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;
import ru.android.zheka.fragment.Home;

@Module
public abstract class HomeBinding {
        @FragmentScope
        @ContributesAndroidInjector(modules = { HomeBindingModule.class
        })
        abstract Home homeFragment();
}

File: routeMap/app/src/main/java/ru/android/zheka/di/HomeBinding.kt
package ru.android.zheka.di

import dagger.Module
import dagger.android.ContributesAndroidInjector
import ru.android.zheka.fragment.Home

@Module
abstract class HomeBinding {
    @FragmentScope
    @ContributesAndroidInjector(modules = [HomeBindingModule::class])
    abstract fun homeFragment(): Home?
}